recipes/statista/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "statista" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MAfarrag/statista/archive/{{ version }}.tar.gz
+  sha256: 6a4b4578e54ad4b4fdff803ff9041a536f744871f7fa926883649319b6f1e52b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python  >=3.7.1,<3.10
+    - pip >=21.3.1
+  run:
+    - python >=3.7.1,<3.10
+    - pip >=21.3.1
+    - matplotlib-base >=3.4.2,<3.6.0
```

```suggestion
    - numpy
    - matplotlib-base >=3.4.2,<3.6.0
```

your package seems to depend on numpy. You should depend on it
https://github.com/MAfarrag/statista/blob/main/statista/tools.py